ParamTypeEnum.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+class ParamType(Enum):
+    PATH = "path"
```

Comment:
Did you mean to use apostrophes here?

Reply:
Did the response to the previous comment answer this question?

Reply:
Yes and if following CKAN this isn't an error message / human-readable string that would ever contain a single quote so it would go in single quotes methinks. Again just a style nitpick